.prospector.yml

```diff
@@ -5,11 +5,12 @@ ignore-paths:
   - _bak
   - _tmp
   - env
+  - venv
   - tests
 
 pylint:
   options:
-    max-attributes: 12
+    max-attributes: 20
```

Comment:
Upping the max attributes to 20 which should be a sane value when dealing with complex Qt window.